packages/common/src/schema.graphql

```diff
@@ -1295,6 +1298,60 @@ type StocktakeNode {
   comment: String
   description: String
   status: StocktakeNodeStatus!
+  lines: StocktakeLinesResponse!
+}
+
+type StocktakeLineNode {
+  id: String!
+  itemName: String!
+  itemCode: String!
+  batch: String
+  expiryDate: NaiveDate
+  snapshotNumPacks: Int
+  snapshotPackSize: Int
+  countedNumPacks: Int
+  sellPricePerPack: Float
+  costPricePerPack: Float
+}
```

Comment:
I think we might need location ( `locationId`? ) and comment and possibly also the `reason` used by Ivory Coast

Reply:
Yup I think there's a fair amount left out .. really just hoping to get all the bulk annoying stuff done so we can make some small more targeted issues